Content.Client/Popups/PopupSystem.cs

```diff
@@ -43,7 +42,10 @@ public void PopupCursor(string message)
 
         public void PopupCoordinates(string message, EntityCoordinates coordinates)
         {
-            PopupMessage(message, _eyeManager.CoordinatesToScreen(coordinates));
+            var mapCoords = coordinates.ToMap(EntityManager);
+            if (_eyeManager.CurrentMap != mapCoords.MapId)
```

Comment:
the client can already see multiple maps at once using viewsubscriptions. i dont have time to figure out how you can check which maps are seen atm but i know this is wrong